### PR TITLE
Jingwei/982 onboarding card compoent

### PR DESCRIPTION
**Pull request by sky1122**
changes:
     1. create OnboardingCard file in website/src/modules
     2. create index.js
     3. create OnboardingCard.js
     4. StyledOnboardingCard.js
 Issue:
    1. import { Typography } from '@devlaunchers/components/components/atoms'  seems I didn't use It correctly, so the font is not working.

How to use the component:
the demo of how to use this component is in user-profile/src/UserOnboardingModal/UserOnboardingModal.js

image

### CODE REVIEW

File: apps/website/src/components/modules/UserOnboardingModal/OnboardingCard/StyledOnboardingCard.js
Comment (judeamos) [inline apps/website/src/components/modules/UserOnboardingModal/OnboardingCard/StyledOnboardingCard.js:7]: you have radius & border-radius used here, should it not be radius?

File: apps/website/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js
Comment (judeamos) [inline apps/website/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js:17]: This "TextWrapper" is not needed here. You can add your paddings to the "PicWrapper" styled component. You can also utilise:

-` gap: 10px;
  gap: 10px 20px; /* row-gap column gap */
  row-gap: 10px;
  column-gap: 20px;` css properties for flex boxes.

File: apps/website/src/components/modules/UserOnboardingModal/OnboardingCard/StyledOnboardingCard.js
Comment (judeamos) [inline apps/website/src/components/modules/UserOnboardingModal/OnboardingCard/StyledOnboardingCard.js:45]: remove console.logs

**Comment by sky1122** [comment]
I make these changes, however, because the <typography> tag in the library cannot change the color. So, for now, leave the font.

**Review by HavreLoic — COMMENTED** [review]
Hey Jingwei there is a merge conflict in apps\user-profile\src\components\modules\UserOnboardingModal\UserOnboardingModal.js I think that once it is solved this branch is good to go

**Comment by sky1122** [comment]
I resolve this conflict, please take a look when you are free.

File: apps/user-profile/src/pages/users/me.js
Comment (judeamos) [inline apps/user-profile/src/pages/users/me.js:21]: revert your changes here. Do not push me.js

Comment (judeamos) [reply apps/user-profile/src/pages/users/me.js:21]: So just uncomment `return !(userData...)` , and delete  the `return true`

File: apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js
Comment (judeamos) [inline apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js:4]: Please can you add a component comment describing what the component does: 
Screenshot 2023-04-23 at 19 37 40

**Review by judeamos — APPROVED** [review]
Just two changes, need you to make. I've approved the PR, once you pushed the changes, you can then merge yourself 👍🏾

**Pull request merged**
